.github/workflows/maven-automatic.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         settings-path: ${{ github.workspace }} # location for the settings.xml file
 
     - name: Build with Maven
-      run: mvn -Drevision=1145.vb_cf6cf6ed960 -B package --file pom.xml
+      run: mvn -Dchangelist=1145.vb_cf6cf6ed960 -Drevision=1145.vb_cf6cf6ed960 -B package --file pom.xml
```

Comment:
Do we still need `-Drevision`?

Reply:
I believe some of the older branches still have the old version of the `pom.xml` file and I thought keeping the `-Drevision` flag could help with possible issues that might come from working on older branches. But since this is only going to run on the Master branch or PRs targetting Master,I think it's not necessary to keep it anymore.

TL;DR: I thought we might need it for older branches but this action only runs on master or things targetting Master so I'll remove it

Reply:
I should add, the `-Drevision` flag might be necessary for older branches if we for some reason need to do a manual release of them, so I'll keep it on the manual action.